### DISCUSSION TITLE
フィードバック一覧

### DISCUSSION THREAD

**Discussion by gassi4731** [lit-kansai/dj_system-docs#52] [ドキュメント]
フィードバックは全部ここに

**Comment by gassi4731** [comment]
Goodというか、まるっと体験できるレベルまで実装できているので、さすがという感じではあります！いいところはたくさんありますが、フィードバックということなので、僕の主観多めで。

- MC: 管理画面かメンバーの共有している画面かパッとわかりづらい
  - 管理画面側はメンバーに見せることを想定している？メッセージもMCが読み上げるスタイルなら特にスクリーンにも映さなくて済むのでは（デザインや管理コストの話です。管理画面はbootstrap程度で機能面よりでよいと思ったので）
- メンバー: 
  - 配布するリンクのトップページの検索窓、ちょっと不親切かな。「入力しよう！」とか言って欲しい。笑 あと、例とかあったらすっと打てる気もする。
  - 自分が投稿したやつの一覧は見たい。複数投稿したいかも。
- 投稿画面: 
  - `+`のボタン、ちょっと違和感あった。（画像の中心にあったほうが、ともおもったけど、カードの中心にしてるんだよね、うーん）
  - これって、最初の20件を取得って感じ？なかったら、検索ワード変える、になる？さらに読み込むとかないのかな、と。
- パフォーマンス
  - 各ページでlighthouseを測ってみるとよいかなと。インデックスされないにしても、参考になる指摘箇所はあるかなと。

ざっとこんな感じを思いました！みずとは寿司屋で、管理画面の話しかしていないので、改めてみてみた！
連続投稿防ぐために、時間制限かけてたんすね、かなりいいなぁと思いました。（なので、複数投稿は無視してくれてもいいかなと）

**Comment by konnyakuken** [comment]
きゃあ。さんからのFBです！

会場に応じてテーマカラーを変えてキャンプごとの特色を出すのなら、リピーターの数やコース、リクエストした曲のジャンルに応じて、Apple Cardみたいにグラデーションを変えるとよさそう！

MCが画像を追加できるなどカスタマイズ性を高めてMCの特色をもっと全面に出せるようにしても面白そうだよね
→画像を入れてクオリティの低下を気にするのなら、それをそのまま使うのではなく水彩画風にするとかにしてみるとか
